src/openms/source/IONMOBILITY/IMDataConverter.cpp

```diff
@@ -185,7 +185,7 @@ namespace OpenMS
       // collapse for scans that actually have a float data array).
       if (in[k].containsIMData())
       {
-        MSExperiment frame = IMDataConverter::splitByIonMobility(in[k], number_of_bins);
+        MSExperiment frame = IMDataConverter::splitByIonMobility(in[k], number_of_bins); // TODO: @cbielow why no move version? e.g., for spec meta values
```

Comment:
@cbielow any comments?

Reply:
true.
one can probably do `std::move(in[k])`.